code/controllers/subsystem/input.dm

```diff
@@ -7,29 +7,36 @@ SUBSYSTEM_DEF(input)
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
 
 	var/list/macro_set
-	var/list/movement_keys
-
 
 /datum/controller/subsystem/input/Initialize()
-	macro_set = list("Any" = "\"KeyDown \[\[*\]\]\"", "Any+UP" = "\"KeyUp \[\[*\]\]\"", "Back" = "\".winset \\\"input.text=\\\"\\\"\\\"\"",
-		"Tab" = "\".winset \\\"input.focus=true?map.focus=true:input.focus=true\\\"\"", "Escape" = "\".winset \\\"input.text=\\\"\\\"\\\"\"",)
-
-	movement_keys = list("W" = NORTH, "A" = WEST, "S" = SOUTH, "D" = EAST,// WASD
-		"North" = NORTH, "West" = WEST, "South" = SOUTH, "East" = EAST)	// Arrow keys & Numpad
+	setup_default_macro_sets()
 
 	initialized = TRUE
 
 	refresh_client_macro_sets()
 
 	return ..()
 
+// This is for when macro sets are eventualy datumized
+/datum/controller/subsystem/input/proc/setup_default_macro_sets()
```

Comment:
Are you sure this is intended, this hard codes in the values for `O L T M`

(Comment was missed from the review, not sure what happened)

Reply:
For now it's how /tg/ is doing it.
Keybinds have some lag, and we want chat to be instant.